src/connection.rs

```diff
@@ -225,6 +225,44 @@ impl Connection {
         Ok(())
     }
 
+
+    #[cfg(feature = "encryption")]
+    #[inline]
+    pub fn encryption_key<T: AsRef<str>>(&self, data: T) -> Result<()> {
+        unsafe {
+            ok!(
+                self.raw.0,
+                ffi::sqlite3_key(
```

Comment:
Should we switch to version 2 perhaps? Seems to be supported by both.

https://www.zetetic.net/sqlcipher/sqlcipher-api/#sqlite3_key